### ISSUE TITLE
gold-cc-expiration-input accessibility issues 

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/gold-cc-expiration-input#5]
Issues with the hint text: 
On Chrome / Mac / VoiceOver: The hint text for MM / YY is reading as "blank". 
On Chrome on Windows with NVDA, it just reads as "edit" without any context of the hint text. 
On Firefox with NVDA, the experience is similar and reads as "edit" but also speaks "has auto complete" which I don't think is true. 

Please note that Chrome OS + ChromeVox is able to read the hint text MM / YY just fine, but this is not the case on other OS / browser / screen reader combos. 

Contrast: the contrast of the hint text for MM / YY is poor. Please make the text a bit darker. 

Audit results: 
[Severe] Controls and media elements should have labels (8)
[Warning] Text elements should have a reasonable contrast ratio (6)
Not applicable tests (9)

**Comment by notwaldorf** [comment]
Done! :tada:
